File: data/src/main/java/com/icehousecorp/maunorafiq/data/city/disk/RealmService.java
package com.icehousecorp.maunorafiq.data.city.disk;

import com.icehousecorp.maunorafiq.data.city.entity.CityEntity;

import java.util.List;

/**
 * Created by maunorafiq on 12/2/16.
 */

public interface RealmService {

    List<CityEntity> get();

    boolean put(String cityName);

    void update(String cityName, int ordinal);
}

File: data/src/main/java/com/icehousecorp/maunorafiq/data/city/disk/RealmService.kt
package com.icehousecorp.maunorafiq.data.city.disk

import com.icehousecorp.maunorafiq.data.city.entity.CityEntity

/**
 * Created by maunorafiq on 12/2/16.
 */

interface RealmService {

    fun get(): List<CityEntity>

    fun put(cityName: String): Boolean

    fun update(cityName: String, ordinal: Int)
}
